packages/react/src/components/stepper/Step.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+
+import { IconCheck, IconError, IconPlaybackPause } from '../../icons';
+// import core base styles
+import 'hds-core';
+import styles from './Stepper.module.scss';
+import classNames from '../../utils/classNames';
+
+export type StepProps = React.ComponentPropsWithoutRef<'button'> & {
+  /**
+   * A custom aria label for selected state of step
+   */
+  customSelectedAriaLabel?: string;
+  /**
+   * Data test id of step
+   */
+  dataTestId?: string;
+  /**
+   * The label of the step
+   */
+  label: string;
+  /**
+   * The language of the step
+   */
+  language?: Language;
+  /**
+   * The number of the step
+   */
+  number: number;
+  /**
+   * A callback function for custom action on step click
+   */
+  onStepClick?: (event, number: number) => void;
+  /**
+   * A function for rendering a custom aria label for step's state
+   */
+  renderCustomStateAriaLabel?: (state) => string;
+  /**
+   * A function for rendering a custom step count label
+   */
+  renderCustomStepCountLabel?: (step, totalNumberOfSteps) => string;
+  /**
+   * A boolean indicating whether step is in selected state
+   */
+  selected?: boolean;
+  /**
+   * A boolean indicating whether small variant is used
+   */
+  small?: boolean;
+  /**
+   * The state of the step
+   */
+  state: 'available' | 'filled' | 'disabled' | 'attention' | 'paused';
+  /**
+   * The total number of steps
+   */
+  stepsTotal: number;
+};
+
+type Language = 'en' | 'fi' | 'sv' | string;
+
+const getStepCountLabel = (language: Language, step: number, totalNumberOfSteps: number) => {
+  return {
+    en: `Step ${step}/${totalNumberOfSteps}.`,
+    fi: `Vaihe ${step}/${totalNumberOfSteps}.`,
+    sv: `Steg ${step}/${totalNumberOfSteps}.`,
+  }[language];
+};
+
+const states = {
+  available: {
+    fi: 'Valittavissa.',
+    en: 'Available.',
+    sv: 'Valbara.',
+  },
+  filled: {
+    fi: 'Täytetty.',
+    en: 'Filled.',
+    sv: 'Fylld.',
+  },
+  disabled: {
+    fi: 'Ei käytettävissä.',
+    en: 'Disabled.',
+    sv: 'Inaktiverad.',
+  },
+  attention: {
```

Comment:
In Attention and Paused steps I think it could be beneficial if the developer could give them a custom or an additional label. This label would be only "visible" for assistive technologies, so maybe something to discuss at the accessibility meeting?

Reply:
This is possible in the current implementation